homework/index.js

```diff
@@ -23,28 +23,73 @@
       if (key === 'text') {
         elem.textContent = value;
       } else {
-        elem.setAttribute(key, value);
+        elem.setAttribute(key, value); // ?
       }
     });
     return elem;
   }
 
+  function changeDateTimeFormat(dateTime) {
+    const timeFormat = new Date(dateTime);
+    return timeFormat.toLocaleString();
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const list = createAndAppend('li', ul, { class: 'topClass' });
```

Comment:
By convention, CSS identifiers should be in **kebab-case**: `top-class`.

Reply:
Hi @remarcmij, thanks for your feedback. 
I used a table as you suggested. Thus, I removed the invalid li tags and also unlinked <a> tags.  To minimize table creating function, I tried to use your hint from other comments about the first row and <a> tag connection.